README.md

```diff
@@ -207,9 +207,15 @@ Looking for conceptual tutorials and explanations? We have some additional artic
 
 &nbsp;
 
-## Pre-training
+## Pretraining
 
-Porting from Lit-LLaMA in progress 👷
+
+
+We provide [a simple training script based on Fabric](pretrain/redpajama.py) if you want to venture into pretraining on RedPajama, a reproduction of the original LLaMA dataset. Conversion scripts for our optimized streaming `PackedDataset`are included. We also have [an alternative RedPajama pretraining script using PyTorch Lightning `Trainer` class](pretrain/redpajama_trainer.py), which has additional features compared to the leaner Fabric implementation.
```

Comment:
redpajama_trainer doesn't exist. We do have `openwebtext_trainer` though

Reply:
good catch, I will include it in another PR when adding docs for openwebtext